front_end/src/actions/types.js

```diff
@@ -13,3 +13,5 @@ export const GET_PROFILES = "GET_PROFILES";
 export const GET_LANDING_DATA = "GET_LANDING_DATA";
 
 export const GET_DELIVERY = "GET_DELIVERY";
+
+export const SEARCH_RESULT_OVERVIEW = "SEARCH_RESULT_OVERVIEW";
```

Comment:
Please leave a comment to label what feature this type is used in.

Reply:
Changes made